docs/self-managed/operational-guides/multi-region/dual-region-ops.md

```diff
@@ -586,13 +570,10 @@ desired={<Ten viewBox="140 40 680 500" />}
 
 <div>
 
-#### Current state
-
-The Camunda components are currently not reachable by end-users and will not process any new process instances. This allows creating a backup of Elasticsearch without losing any data.
-
-#### Desired state
-
-You are creating a backup of the main Elasticsearch instance in the surviving region and restore it in the recreated region. This Elasticsearch backup contains all the data and may take some time to be finished.
+| **Details**              | **Current State**                                                                                                        | **Desired State**                                                                                                                                                |
+| ------------------------ | ------------------------------------------------------------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| **Camunda Components**   | Not reachable by end-users and not processing any new process instances. This state allows for data backup without loss. | Creating a backup of the main Elasticsearch instance in the surviving region and restoring it in the recreated region. Backup process may take time to complete. |
```

Comment:
I'm not sure the desired state brings much value since it's the same as below for the Elasticsearch Backup.

The bit about
> Backup process may take time to complete.
could be added to the desired state below.